exporters/jaeger/src/main/java/io/opentelemetry/exporter/jaeger/MarshalerCollectorServiceGrpc.java

```diff
@@ -42,19 +42,17 @@ public PostSpansResponse parse(InputStream stream) {
         }
       };
 
-  private static volatile MethodDescriptor<PostSpansRequestMarshaler, PostSpansResponse>
-      getPostSpansMethod;
+  private static final MethodDescriptor<PostSpansRequestMarshaler, PostSpansResponse>
```

Comment:
The lazy technique I had copied from gRPC generated code was presumably for reasons related to having lots of protobufs in a single app. We don't have such an issue to require laziness I guess